packages/mdc-button/mdc-button.scss

```diff
@@ -65,8 +65,10 @@
     }
   }
 
-  @each $theme-style in (primary, accent) {
-    &.mdc-button--#{$theme-style} {
+  @each $theme-style in (primary, secondary) {
+    $modifier: if($theme-style == "secondary", "accent", $theme-style);
```

Comment:
Add a comment about why this is around...basically about backwards compatibility